regolith/schemas.py

```diff
@@ -3620,6 +3621,172 @@
         },
     },
+    "projecta": {
+        "_description": {
+            "description": "This collection describes a single deliverable "
+                            "of a larger project."
+        },
+        "_id": {
+            "description": "Unique projecta identifier",
+            "required": True,
+            "type": "string"
+        },
+        "begin_date": {
+            "description": "projecta start date, yyyy-mm-dd",
+            "required": False,
+            "type": "string"
+        },
+        "collaborators": {
+            "description": "list of collaborator's id's",
+            "required": False,
+            "type": "list"
+        },
+        "deliverable":{
+            "description": "outline of the deliverable for this projecta",
+            "type": "dict",
+            "schema": {
+                "audience": {"description": "who is this deliverable for",
+                             "required": False,
+                             "type": "list"},
+                #is this the proper form of the due date, the exemplar is ambiguous
+                "due_date": {"description": "due date of deliverable, yyyy-mm-dd",
+                             "required": False,
+                             "anyof_type": ["date", "string"]},
+                "success_def": {"description": "definition of a successful deliverable",
+                                "required": False,
+                                "type": "string"},
+                "scope": {"description": "",
+                          "required": False,
+                          "type": "list"},
+                "platform": {"description": "description of how and where the audience will access "
+                                "the deliverable.  e.g. Journal if it is a paper",
+                             "required": False,
+                             "type": "string"},
+                "rollout": {"description": "steps that the audience will take to access "
+                                           "and interact with the deliverable, "
+                                           "not needed for paper submissions",
+                            "required": False,
+                            "type": "list"},
+                "notes": {"description": "deliverable note",
+                          "required": False,
+                          "type": "list"},
+                "status": {"description": "current state of deliverable",
+                           "required": False,
+                           "type": "string",
+                           "eallowed": PROJECTA_STATUS},
+            }
+        },
+        "description": {
+            "description": "explanation of projecta",
+            "required": False,
+            "type": "string"
+        },
+        "end_date": {
+            "description": "projecta end date, yyyy-mm-dd. "
+                           "Not necessarily the same as deliverable due date",
```

Comment:
you can delete this line.  Due dates and end dates are never the same. (unless the thing happens to actually be delivered when it was planned to be.....).   btw, I think this may be a concept missing from Yevgeny's mosy schema, but I let him take the lead on this.....each action on the action graph should have a start and an end datetime that records when it took place, but for planning purposes (when it is recipe and not a dish) it is helpful to have a projected start and end time, or maybe more useful is a due-time and a duration.  I could imagine in the future that if actions had durations and sequence we could find algorithms that would build optimal plans (our beloved gantt charts)